README.md

```diff
@@ -12,60 +12,58 @@ platform.
 
```

Comment:
above line: "you can push notification to" -> "you can send push notifications to any"

"for server-side notification" -> "for server side notifications"